packages/blocks/src/database-block/table/components/column-type/select/multi-tag-view.ts

```diff
@@ -40,12 +40,27 @@ export class MultiTagView extends WithDisposable(ShadowlessElement) {
     }
   `;
 
+  @query('.affine-database-select-cell-container')
+  selectContainer!: HTMLElement;
+
   @property()
   value: string[] = [];
 
   @property()
   options: SelectTag[] = [];
 
+  @property()
+  setHeight: ((height: number) => void) | undefined;
```

Comment:
Could you consider using the  `setHeight?` or `setHeight!` style? Our code style is trying the best avoid writing the `undefined` literal.